ajs-upgraded-providers.ts

```diff
@@ -102,32 +102,6 @@ export const HsDimensionServiceProvider = {
   deps: ['$injector'],
 };
 
-/**
- * @param i
- */
-export function hsQueryBaseService(i: any) {
-  return i.get('HsQueryBaseService');
-}
-
-export const HsQueryBaseServiceProvider = {
-  provide: HsQueryBaseService,
```

Comment:
You can safely remove the imports of `HsQueryBaseService` and `HsQueryVectorService` from the top of the file.

Reply:
Done